### PR TITLE
NOISSUE - Fix failing CI

### PR DESCRIPTION
**Pull request by felixgateru**
# What type of PR is this?
This is a refactor as it makes changes to the test files to ensure all tests are pasiing.

## What does this do?
Cleans up code to ensure alll tests are passing

## Which issue(s) does this PR fix/relate to?

- Related Issue #2443

### CODE REVIEW

File: internal/groups/api/endpoint_test.go [hunk 1]
```diff
@@ -160,20 +166,27 @@ func TestViewGroupEndpoint(t *testing.T) {
...
 			svcErr:  nil,
 			resp:    viewGroupRes{},
 			err:     svcerr.ErrAuthorization,
 		},
+		{
+			desc:    "unsuccessfully with invalid request",
+			session: pauth.Session{DomainUserID: validID, UserID: validID, DomainID: validID},
+			req:     groupReq{},
+			svcResp: validGroupResp,
+			svcErr:  nil,
```
Comment (nyagamunene) [inline internal/groups/api/endpoint_test.go]: This not being used.

File: pkg/sdk/go/tokens_test.go [hunk 2]
```diff
@@ -106,6 +109,7 @@ func TestIssueToken(t *testing.T) {
 			svcCall := svc.On("IssueToken", mock.Anything, tc.login.Identity, tc.login.Secret, tc.login.DomainID).Return(tc.svcRes, tc.svcErr)
 			resp, err := mgsdk.CreateToken(tc.login)
 			assert.Equal(t, tc.err, err)
+			fmt.Println("err", err)
```
Comment (nyagamunene) [inline pkg/sdk/go/tokens_test.go]: Remove this log

File: pkg/sdk/go/tokens_test.go [hunk 3]
```diff
@@ -157,34 +162,35 @@ func TestRefreshToken(t *testing.T) {
...
+			err:      errors.NewSDKErrorWithStatus(apiutil.ErrBearerToken, http.StatusUnauthorized),
 		},
 	}
 	for _, tc := range cases {
 		t.Run(tc.desc, func(t *testing.T) {
-			svcCall := svc.On("RefreshToken", mock.Anything, tc.token, tc.login.DomainID).Return(tc.svcRes, tc.svcErr)
+			authCall := auth.On("Identify", mock.Anything, mock.Anything).Return(&magistrala.IdentityRes{Id: validID, UserId: validID, DomainId: validID}, tc.identifyErr)
+			svcCall := svc.On("RefreshToken", mock.Anything, pauth.Session{DomainUserID: validID, UserID: validID, DomainID: validID}, tc.token, tc.login.DomainID).Return(tc.svcRes, tc.svcErr)
 			resp, err := mgsdk.RefreshToken(tc.login, tc.token)
+			fmt.Println("err", err)
```
Comment (nyagamunene) [inline pkg/sdk/go/tokens_test.go]: same

**Review by nyagamunene — APPROVED** [review]
LGTM

**Review by arvindh123 — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**